### ISSUE TITLE
FLUX.1 LoRA Training Not Converging

### ISSUE THREAD

**Issue by Xayah-Hina** [bghira/SimpleTuner#2086]
Hello! Thank you for the awesome project.

I followed the FLUX instructions in the WebUI to train a very simple LoRA for several thousand iterations, but it doesn’t seem to be converging anymore. Would it be possible to provide more detailed guidance or a way to check whether the training is progressing correctly?

Thanks!

**Comment by bghira** [comment]
you mean it is like the model got stuck ? it could be too low of a learning rate or too high of a batch size (they're related).

can you share more details on the failure mode and what you're expecting to happen

**Comment by bghira** [comment]
i was trying z-image with domokun dataset and saw this after 2900 steps

<img width="2054" height="1071" alt="Image" src="https://github.com/user-attachments/assets/9b58b311-7c10-4b5b-9842-dcd3f49af36f" />

however, that was with a flow schedule shift of 0.78

when i set it to 3.0 i see this after 600 steps

<img width="2054" height="1071" alt="Image" src="https://github.com/user-attachments/assets/083341fc-5347-4588-9354-973f63a617a4" />
